packages/service-clients/end-to-end-tests/azure-client/src/test/AzureClientFactory.ts

```diff
@@ -88,7 +97,7 @@ export function createAzureClient(
 			}
 		: {
 				tokenProvider: new InsecureTokenProvider("fooBar", user, scopes),
-				endpoint: "http://localhost:7071",
+				endpoint: "http://localhost:7071", // Port for local Azure Fluid Relay (AFR) service
```

Comment:
```suggestion
				endpoint: "http://localhost:7071",
```
It's really for tinylicious, not sure the new comment adds much. Same on lines 167 and 216 below, I'd remove it.